apps-api/ballot-crypto/src/modules/key/services.ts

```diff
@@ -0,0 +1,113 @@
+import { InternalErrorCode } from '@pple-today/api-common/dtos'
+import { ElectionKeysStatus } from '@pple-today/database/prisma'
+import Elysia from 'elysia'
+import { ok } from 'neverthrow'
+
+import {
+  BackofficeAdminService,
+  BackofficeAdminServicePlugin,
+} from '../../plugins/backoffice.admin'
+import { KeyManagementPlugin, KeyManagementService } from '../../plugins/kms'
+import { mapGoogleAPIError } from '../../utils/error'
+
+export class KeyService {
+  private readonly MAX_RETRY = 5
```

Comment:
We don't quite sure that when key turns to `ENABLED` state, please extends `MAX_RETRY` to 10 to give them another chances 😅